src/docs/product/discover-queries/query-builder/query-equations.mdx

```diff
@@ -0,0 +1,141 @@
+
+If you’re interested in being an Early Adopter, you can turn your organization’s Early Adopter status on/off in General Settings. This will affect all users in your organization and can be turned back off just as easily.
+
+</Note>
+
+In **Discover**, you can create equations based on the columns of your query. Each column acts like a variable since it represents the result of the query, which is a number. This number is used in the equation.
+
+For example, let's take an equation that calculates the ratio of a transaction's first contentful paint (`measurements.fcp`) to the transaction's largest contentful paint (`measurements.lcp`):
+
+equation = `measurements.fcp / measurements.lcp`
+
+The equation will take the result of each of those query columns and use those numbers in the calculation. In this case, the results of those query columns are:
+
+- measurements.fcp = 50
+- measurements.lcp = 100
+
+So the equation becomes:
+
+`50 / 100`
+
+The result of the equation is then displayed in the updated table as shown in the recording below:
+
+![Adding an equation in discover](./add_equation.gif)
+
+These equations can help you calculate things like:
+
+- [The percentage of transactions that completed within a threshold](#calculate-a-status-percentage)
+- [A custom Apdex score](/#calculate-a-custom-apdex)
+- [The average non-database times](#find-largest-average-non-database-times)
+
+To add an equation to a query:
+
+1. Click the "Columns" button to open the column editor.
+2. If they aren't already included in your query, add the columns you need for your equation.
+3. Click "Add an Equation".
+4. Enter your equation by selecting columns, entering numbers (if needed), and adding operators.
+5. Click "Apply".
+
+## Guidelines for Equations
+
+Equations must:
+
+- Contain at least one field or function
+- Contain at least one operator
+- Have explicit operators. For example, `a(a+b)` is invalid; instead, you should enter `a * (a + b)`
+
+Equations cannot:
+
+- Contain a mix of fields or functions
+- Contain exponents which aren't supported
+
+Equations will follow expected order of operations, resolving brackets first, division and multiplication next followed by addition and subtraction. Equations can also include negative numbers.
+
+Equations can only operate on columns and functions that are numbers, which includes:
+
+- Functions:
+    - count
+    - count_unique
+    - count_if
+    - failure_count
+    - avg
+    - sum
+    - percentiles (p50, p75 etc.)
+    - apdex
+    - user_misery
+    - eps() and epm()
+- Fields
+    - transaction.duration
+    - measurements.fp
+    - measurements.fcp
+    - measurements.lcp
+    - measurements.fid
+    - measurements.cls
+    - measurements.ttfb
+    - spans.browser
+    - spans.http
+    - spans.db
+    - spans.resource
+
+## Example Equations
+
+### Calculate a status percentage
```

Comment:
This title isn't super clear. If you update the heading, don't forget to update the anchored link as well.

```suggestion
### Calculate the percentage of transactions completed
```